__tests__/nameit.tests.ps1

```diff
@@ -2,6 +2,7 @@ $manifestPath = $PSScriptRoot | Join-Path -ChildPath '..' | Join-Path -ChildPath
 Import-Module $manifestPath -Force
 
 Describe "NameIT Tests" {
+    $StaticSeed = [System.Random]::new().Next()
```

Comment:
Originally chose a constant magic seed, but this kind of screws up `Get-Random` for everything that runs in the same session after the tests run. Fine for AppVeyor, not so great for your dev machine. Decided instead to randomly produce a new seed at the beginning of the tests to use throughout testing. Should retain decent pseudorandomness post tests.